jetty-server/src/main/config/etc/jetty-threadlimit.xml

```diff
@@ -10,7 +10,7 @@
     <Arg>
       <New id="ThreadLimitHandler" class="org.eclipse.jetty.server.handler.ThreadLimitHandler">
         <Arg name="forwardedHeader"><Property name="jetty.threadlimit.forwardedHeader"/></Arg> 
-        <Set name="enabled" property="jetty.threadlimit.enabled"/> 
+        <Set name="enabled"><Property name="jetty.threadlimit.enabled" default="true"/></Set>
```

Comment:
Was not enabled before, why was changed to enabled?

Reply:
Because if its not enabled it's not doing anything, and the value previously was `true`, so I've reinstated it. The value in the class is `false`. Again, see commit https://github.com/eclipse/jetty.project/commit/343cf732b9a5aacd4c84fcb937a4b1624a1a0afd

Reply:
Again, breaks the behavior of all Jetty 10 releases.
If it's a wrong value, let's open a new issue rather than doing it here where it gets lost in other changes.